ui/admin/app/templates/scopes/index.hbs

```diff
@@ -19,16 +18,12 @@
       </main>
     </sidebarLayout.body>
     <sidebarLayout.footer>
-      <Rose::Footer as |footer|>
-        <footer.text>{{company-copyright}}</footer.text>
-        <footer.text>{{app-name}}</footer.text>
-        <footer.nav as |nav|>
-          <nav.link @href={{(doc-url)}}>
-            {{t 'titles.documentation'}}
-          </nav.link>
-        </footer.nav>
-        <footer.brand @logo='logo-company' @text={{(company-name)}} />
-      </Rose::Footer>
+      <Hds::AppFooter @theme={{this.session.data.theme}} as |AF|>
+        <AF.Link
+          @href={{(doc-url)}}
+        >{{t 'titles.documentation'}}</AF.Link>
+        <AF.LegalLinks />
+      </Hds::AppFooter>
```

Comment:
@DhariniJeeva I wasn't able to figure out how to view this page in the browser. Could you tell me how to navigate to it? I think I need to add controller for it too but didn't want to add it blindly.

Reply:
Lisbet helped me find it so I've now updated it also. :)